resources/lang/en/messages.php

```diff
@@ -1,11 +1,14 @@
 <?php
 
 return [
-    "error_create_admin"       => "Wrong count\'t created admin user, please try againt!",
-    "error_create_city"        => "Wrong count\'t created city, please try againt!",
+    "error_create_admin" => "Wrong! couldn't create admin user, please try againt.",
```

Comment:
[rank A] I think it is "again", not "againt". Please check English spelling carefully.